### DISCUSSION TITLE
How to setup Medusa monorepo properly ?

### DISCUSSION THREAD

**Discussion by minhchu** [medusajs/medusa#7365] [Q&A] [answered]
I set up the Medusa repo following this guide: https://docs.medusajs.com/development/fundamentals/local-development. However, there are errors and I don't know how to fix them. Does anyone have any ideas on how to fix this? Thank you.

<img width="1098" alt="SCR-20240519-bcab-2" src="https://github.com/medusajs/medusa/assets/14136809/6f99ddd5-d86f-4b04-93a1-aa3561d160fc">

**Comment by TuanAnhQy97** [comment, marked as answer]
You should run this command first `npx turbo run build`
